src/apps/explorer/layout/Header.tsx

```diff
@@ -29,28 +30,6 @@ const Logo = styled.span`
   }
 `
 
-const NetworkLabel = styled.span`
-  border-radius: 0.6rem;
-  display: flex;
-  margin: 0 0.5rem;
-  font-size: 1.1rem;
-  text-align: center;
-  padding: 0.7rem;
-  text-transform: uppercase;
-  font-weight: ${({ theme }): string => theme.fontBold};
-  letter-spacing: 0.1rem;
-
-  &.rinkeby {
-    background: ${({ theme }): string => theme.borderPrimary};
```

Comment:
The color for Rinkeby could be: #f6c343 and text in a dark color
![image](https://user-images.githubusercontent.com/622217/145639696-f2ac4c79-b81b-4d0d-83c0-9e49e573eb04.png)

Based on MM (as Anxo suggested)
![image](https://user-images.githubusercontent.com/622217/145639328-e8937faa-e7ea-4023-b4bb-4744f76a74e5.png)